.buildkite/scripts/build_pr_setup.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# The reminder of this hook should only be invoked for builds triggered by the Buildkite PR bot
+if [ -z ${GITHUB_PR_OWNER+set} ] || [ -z ${GITHUB_PR_REPO+set} ] || [ -z ${GITHUB_PR_TRIGGERED_SHA+set} ];then
+  exit 0
+fi
+
+gitHubToken=$(vault read -field=value secret/ci/elastic-docs/docs_preview_cleaner)
+
+githubPublishStatus="https://api.github.com/repos/${GITHUB_PR_OWNER}/${GITHUB_PR_REPO}/statuses/${GITHUB_PR_TRIGGERED_SHA}"
+data='{"state":"pending","target_url":"'$BUILDKITE_BUILD_URL'","description":"Build started.","context":"buildkite/'$BUILDKITE_PIPELINE_SLUG'"}'
+echo "Setting buildkite/docs commit status to pending"
+curl -s -L \
+  -X POST \
+  -H "Accept: application/vnd.github+json" \
+  -H "Authorization: Bearer ${gitHubToken}" \
```

Comment:
There's not a chance of this getting logged anywhere, is there?

Reply:
I can't think of any other ways to make that call. we could add an explicit set +x just before it?